angular.json

```diff
@@ -166,12 +166,9 @@
           }
         },
         "lint": {
-          "builder": "@angular-devkit/build-angular:tslint",
+          "builder": "@angular-devkit/build-angular:lint",
           "options": {
-            "tsConfig": [
-              "projects/angular-jwt/tsconfig.lib.json",
-              "projects/angular-jwt/tsconfig.spec.json"
-            ],
+            "tsConfig": "e2e/tsconfig.e2e.json",
```

Comment:
I dont think this should have a `tsConfig`; it should also include the files from `project/angular-jwt`, basically the lint config above should be here, and the config above should lint `src`.